content/cpp/concepts/strings/terms/strcmp/strcmp.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: 'strcmp()'
+Description: 'Compares two C-style strings.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Data Types'
+  - 'Characters'
+  - 'Arrays'
+  - 'Functions'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`strcmp()`** string function compares two C-style strings and returns an integer value.
+
+> **Note:** `strcpy()` only works on C-style strings, not C++ style strings. So it only works with strings of type `char str[];` not `string str;`
+
+## Syntax
+
+The `string.h` header file needs to be included at the top of the file:
+
+```cpp
+#include <string.h>
+```
+
+The `strcmp()` function compares C-style strings `string1` and `string2` and returns an integer value.
+
+```pseudo
+strcmp(string1, string2);
+```
+
+- If the two strings are the same, it returns 0.
+- If `string1` > `string2`, it returns a positive value.
+- If `string1` < `string2` or `string1` is a substring of `string2`, it returns a negative value.
+
+## Example
```

Comment:
I think this section could be turned into a Codebyte section since C++ is a supported language.